### PR TITLE
Integration tests for the basic FDW implementation

### PR DESCRIPTION
**Pull request by stillinbeta**

### CODE REVIEW

**Comment by stillinbeta** [comment]
@bluejekyll have you seen this error before? I'm scratching my head over here.

**Comment by bluejekyll** [comment]
Yes! But I’m trying to remember what the issue was 🤔

**Comment by bluejekyll** [comment]
Oh. Take a look at the integration tests. They’re all ignored, because I never figured this out. I’m pretty sure they work locally. I tried many things, like changing the permissions on the shared library, etc.

I’m starting to be reminded by why it’s disabled on travis. The only guess I’ve come up with is that pg tries to copy the shared library into some temp directory and that fails (though that’s not what the error implies).

**Comment by bluejekyll** [comment]
Looking at the source, I remember trying to track this down before. And it does seem to be an issue with finding the library. Is it possible the postgres daemon is being chrooted?

**Comment by stillinbeta** [comment]
@bluejekyll I don't know exactly what the issue is, but this approach (copying to a textfile) seems to work! I didn't notice the `integration-tests` package, but I can move my code into there tomorrow.

File: .travis.yml [hunk 1]
```diff
@@ -46,12 +46,13 @@ matrix:
            RUST_BACKTRACE=full
            POSTGRES_TEST_DB=travis_ci_test
            POSTGRES_USER=postgres
+           POSTGRES_URL=postgres://${POSTGRES_USER}@localhost:5432/${POSTGRES_TEST_DB}
       before_install:
         - rustup component add rustfmt
-        # - brew upgrade postgresql || brew install postgres
```
Comment (stillinbeta) [inline .travis.yml]: this upgrade takes forever and doesn't seem to make much difference in the end.

**Comment by bluejekyll** [comment]
This is good news. Maybe perms on the directory were bad. Dunno. I guess putting it in /tmp gets around that. Probably will fix the integration tests as well.

**Comment by stillinbeta** [comment]
@bluejekyll I think we're good to go!

**Pull request merged**